local/jimm/qa-lxd-mac-forward.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# This script forwards JIMM and Keycloaks ports to localhost from the QA VM.
+#
+# It can be used in a number of ways, as follows:
+# ./qa-lxd-mac-forward.sh jimm2 --linux --wait  - Run for a VM named jimm2, on linux, and wait after tunnels are established
+# ./qa-lxd-mac-forward.sh jimm2 --linux         - Run for a VM named jimm2, on linux, and exit immediately
+# ./qa-lxd-mac-forward.sh --linux               - Run for the default VM (jimm), on linux, and exit immediately
+# ./qa-lxd-mac-forward.sh jimm2 --mac --wait    - Run for a VM named jimm2, on mac, and wait after tunnels are established
+
```

Comment:
Always a good idea to add `set -euo pipefail` at the top to ensure errors stop the script.

Reply:
True, I'll add Wednesday (this applies to all comments I'll do them Wednesday)